src/plugins/data_source_management/public/types.ts

```diff
@@ -136,35 +129,14 @@ export const credentialSourceOptions = [
   sigV4CredentialOption,
 ];
 
-export interface DataSourceAttributes extends SavedObjectAttributes {
-  title: string;
-  description?: string;
-  endpoint?: string;
-  dataSourceVersion?: string;
```

Comment:
This may break the datasource metadata fetch flow, need to double check.

Reply:
I actually tested. I have attached video in detail. `.kibana` index has data source version, installed plugin version information.